_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,120 @@
+---
+id: b4f0c3
+name: meta viewport
+rule_type: atomic
+description: |
+  This rule checks that the meta element is not used to block the user agent ability to zoom. 
+accessibility_requirements:
+  wcag20:1.4.10: # Reflow (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to every `<meta name="viewport">` elements with a `content` attribute in a document.
+
+## Expectation
+
+The `content` attribute of the test target: 
+- does not have the property `user-scalable` or it is set to `yes`and
```

Comment:
To avoid messing up the double logical connector, I would use an Oxford-comma (or rather an Oxford-colon):
"- does not have the property `user-scalable` or it is set to `yes`; and"
Maybe even making it more explicit with an either…or:
"- either does not have the property `user-scalable` or has it set to `yes`; and"

Reply:
Agree with @Jym77 suggestion (I prefer the second wording)